src/main/java/frc/robot/subsystems/swerve/swervestatehelpers/AimAssistUtils.java

```diff
@@ -0,0 +1,42 @@
+package frc.robot.subsystems.swerve.swervestatehelpers;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.kinematics.ChassisSpeeds;
+import frc.robot.subsystems.swerve.SwerveConstants;
+import frc.robot.subsystems.swerve.SwerveMath;
+
+import java.util.function.Supplier;
+
+import static frc.robot.subsystems.swerve.SwerveMath.getDriveMagnitude;
+
+public class AimAssistUtils {
+
+    public static ChassisSpeeds getRotationAssistedSpeeds(
+            ChassisSpeeds speeds,
```

Comment:
```suggestion
            ChassisSpeeds wantedSpeeds,
```

Reply:
no, the speeds is the joystick's speeds and the current speeds is the chassis speeds

Reply:
because you calculate the magnitude on the actual speeds

Reply:
i can change it so that i calculate the magnitude on the wanted speeds

Reply:
the speeds can any wnated speed no matter where i get it from, the name should say what the usage and not want to insert in util stuff